deploy/examples/005_advanced_config_example_cr.yaml

```diff
@@ -16,7 +16,7 @@ spec:
     deploymentStrategy: RollingUpdate
     skipSetup: false
     externalAccess:
-      configHostname: example-quayecosystem-quay-enterprise.apps
+      hostname: example-quayecosystem-quay-enterprise.apps
```

Comment:
Does this require a change? `configHostname` is correct for specifying the location for the Config Route

Reply:
You are right, I will fix that and resubmit!